### PR TITLE
ARISTOTLE: solve some corner cases in reading station data

### PR DESCRIPTION
**Pull request by ptormene**
I would like to provide more information to the webui user in a separate PR, but for now I am aiming at fixing the broken tests and better handling cases in which seismic stations are discarded.

Running aristotle tests locally, only one error remains, for usgs_id `us70008dx7`:

```python
Traceback (most recent call last):                                                                                                                                                                                                           
  File "/home/ptormene/GIT/oq-engine/openquake/engine/aristotle.py", line 217, in main_cmd                                                                                                                                                   
    engine.run_jobs([job])                                                                                                                                                                                                                   
  File "/home/ptormene/GIT/oq-engine/openquake/engine/engine.py", line 404, in run_jobs                                                                                                                                                      
    run_calc(jobctx)                                                                                                                                                                                                                         
  File "/home/ptormene/GIT/oq-engine/openquake/engine/engine.py", line 269, in run_calc                                                                                                                                                      
    calc.run(shutdown=True)                                                                                                                                                                                                                  
  File "/home/ptormene/GIT/oq-engine/openquake/calculators/base.py", line 288, in run                                                                                                                                                        
    raise exc from None                                                                                                                                                                                                                      
  File "/home/ptormene/GIT/oq-engine/openquake/calculators/base.py", line 271, in run                                                                                                                                                        
    self.pre_execute()                                                                                                                                                                                                                       
  File "/home/ptormene/GIT/oq-engine/openquake/calculators/event_based_risk.py", line 347, in pre_execute                                                                                                                                    
    super().pre_execute()                                                                                                                                                                                                                    
  File "/home/ptormene/GIT/oq-engine/openquake/calculators/base.py", line 644, in pre_execute                                                                                                                                                
    calc.run(remove=False)
  File "/home/ptormene/GIT/oq-engine/openquake/calculators/base.py", line 288, in run
    raise exc from None
  File "/home/ptormene/GIT/oq-engine/openquake/calculators/base.py", line 272, in run
    self.result = self.execute()
  File "/home/ptormene/GIT/oq-engine/openquake/calculators/event_based.py", line 695, in execute
    smap = starmap_from_rups(event_based, oq, self.full_lt, self.sitecol, dstore)
  File "/home/ptormene/GIT/oq-engine/openquake/calculators/event_based.py", line 345, in starmap_from_rups
    raise ValueError(
ValueError: The calculation is too large: G=15, M=4, N=4603. You must reduce the number of sites i.e. enlarge region_grid_spacing)
```

### CODE REVIEW

File: openquake/hazardlib/shakemap/parsers.py [hunk 1]
```diff
@@ -386,12 +390,32 @@ def download_station_data_file(usgs_id):
...
+                                f' {seismic_len} seismic stations were all'
+                                f' discarded')
+                        else:
+                            # TODO: return this info also to the webui
+                            logging.warning(
+                                f'{original_len} stations were found, but none'
+                                f' of them are seismic')
+                    else:
+                        # TODO: return this info also to the webui
+                        logging.warning('No stations were found')
```
Comment (micheles) [inline openquake/hazardlib/shakemap/parsers.py:418]: This should be an `info` not a `warning` since it is a quite common case.

**Review by micheles — APPROVED** [review]

File: openquake/hazardlib/shakemap/parsers.py (hunk 1 above)
Comment (ptormene) [reply openquake/hazardlib/shakemap/parsers.py:418]: Ok. I am doing this change in the next PR that I am opening now
